src/interface.jl

```diff
@@ -93,7 +93,7 @@ function transform(t::Transform, x)
     res = with_logabsdet_jacobian(t, x)
     if res isa ChangesOfVariables.NoLogAbsDetJacobian
         error(
-            "`transform` not implemented for $(typeof(f)); implement `transform` and/or `with_logabsdet_jacobian`.",
+            "`transform` not implemented for $(typeof(t)); implement `transform` and/or `with_logabsdet_jacobian`.",
```

Comment:
This change has nothing to do with this PR, I just spotted the typo while working on this PR and didn't feel like making a separate one character PR.